src/psyclone/tests/psyir/symbols/symbol_test.py

```diff
@@ -294,7 +315,7 @@ def test_symbol_copy():
     assert new_sym.visibility == asym.visibility
     # Check that we can modify the interface of the new symbol without
     # affecting the original.
-    new_sym.interface._container_symbol = ContainerSymbol("other_mod")
+    new_sym.interface.container_symbol = ContainerSymbol("other_mod")
     assert asym.interface.container_symbol is csym
 
 
```

Comment:
Since we're here, would you mind fixing the 'f-string' pylint complaint at L363?

Reply:
Done